runner.py

```diff
@@ -0,0 +1,43 @@
+from DataFetch import Data_Fetch
+from UrlGenerator import Url_Generator
+from SummaryGenerator import Summary_Generator
+# from pdf-writer import Pdf_Writer
+# from email-sender import Email_Sender
+
+
+if __name__ == "__main__":
+
+    key = '431fd6ac7c624e3fbe99eb02a13fb35f'
```

Comment:
```suggestion
    key = 'INPUT A KEY'
```

read key from an encrypted static file like .json, or .key